runtime/core/decoder/asr_decoder.cc

```diff
@@ -107,6 +107,10 @@ DecodeState AsrDecoder::AdvanceDecoding(bool block) {
   std::vector<std::vector<float>> ctc_log_probs;
   model_->ForwardEncoder(chunk_feats, &ctc_log_probs);
   int forward_time = timer.Elapsed();
+  for (int i = 0; i < ctc_log_probs.size(); i++) {
```

Comment:
``` cpp 
if (opts_.ctc_wfst_search_opts.blank_scale != 1.0) {
 for (int i = 0; i < ctc_log_probs.size(); i++) {
      ctc_log_probs[i][0] = ctc_log_probs[i][0]
              + std::log(opts_.ctc_wfst_search_opts.blank_scale);
  }
}
```

Reply:
@simonwang517 please fix the logic as above to avoid useless computation when blank_scale is 1.0.

Reply:
@robin1001 have updated code to avoid useless computation